src/drafts/draftsActions.js

```diff
@@ -12,3 +12,14 @@ export const draftRemove = (narrow: Narrow): DraftRemoveAction => ({
   type: DRAFT_REMOVE,
   narrow,
 });
+
+export const updateDraft = (narrow: Narrow, message: string) => (
+  dispatch: Dispatch,
+  getState: GetState,
+) => {
+  if (message.trim().length === 0) {
```

Comment:
Just a thought: I guess we can also remove `draftRemove` and `draftAdd`, and completely rely on `draftUpdate`. 

And in `reducers`, if `message.trim().length === 0` then return state which does't have that `narrow` key in `drafts` else update drafts.

Basically moving this if condition to `reducers`.

Thoughts?

Reply:
I agree -- this logic feels like it'd be cleanest to push inside the reducer, and have just one action type like `DRAFT_UPDATE`. Not a blocker for this branch, though; it's already progress to move it out of ComposeBox.

Ideally the "skip if no change" would live in the reducer too -- after all, that's an important part of every reducer's job already. Does this logic really cut down that much on how many actions we end up dispatching? I don't see why we'd be getting often to `updateDraft` when the text hasn't changed.